CHANGELOG.md

```diff
@@ -51,6 +51,9 @@ Versioning](https://semver.org/spec/v2.0.0.html).
   following fields set to `None`:
   - `Setting::{piglet, reconverge, hog}::rpc`
   - `Setting::reconverge::public`
+- Use `StringNumber` for integers beyond `i32`
+  - In `BlockListSmb`, `Network`, `SamplingPolicy`, `Tidb`, and `TidbRule`, changed GraphQL APIs to return 
+    `StringNumber` or `ID` instead of integers beyond `i32`.
```

Comment:
Because the first line doesn't mention `ID` and the second includes the first , the first doesn't look necessary to me.